src/main/java/uk/gov/hmcts/divorce/common/model/access/DefaultAccess.java

```diff
@@ -11,13 +11,15 @@
 import static uk.gov.hmcts.divorce.common.model.UserRole.CASEWORKER_DIVORCE_COURTADMIN_LA;
 import static uk.gov.hmcts.divorce.common.model.UserRole.CASEWORKER_DIVORCE_SOLICITOR;
 import static uk.gov.hmcts.divorce.common.model.UserRole.CASEWORKER_DIVORCE_SUPERUSER;
+import static uk.gov.hmcts.divorce.common.model.UserRole.CASEWORKER_DIVORCE_SYSTEMUPDATE;
 import static uk.gov.hmcts.divorce.common.model.UserRole.CITIZEN;
 
 public class DefaultAccess implements HasAccessControl {
     @Override
     public SetMultimap<HasRole, Permission> getGrants() {
         SetMultimap<HasRole, Permission> grants = HashMultimap.create();
         grants.putAll(CITIZEN, Permissions.CREATE_READ_UPDATE);
+        grants.putAll(CASEWORKER_DIVORCE_SYSTEMUPDATE, Permissions.CREATE_READ_UPDATE);
```

Comment:
why do we need this role to be added ?

Reply:
The system caseworker user who is invoking the linking for joint application does not have the CRU permissions for all Casefields so CCD Data Store was rejecting the request. We need the system case worker to have the same access to the citizen case fields as a citizen user.